crowbar_framework/app/models/batch/export.rb

```diff
@@ -46,7 +46,7 @@ def filename
 
     def persist!
       path.open("w") do |file|
-        file.write content.stringify_keys(true).to_yaml
+        file.write content.stringify_keys.to_yaml
```

Comment:
@MaximilianMeister: Just to help me understand this, this will stringify only top level keys. Why drop the `(true)` argument?